examples/latte/train.py

```diff
@@ -53,6 +53,7 @@ def init_env(
     distributed: bool = False,
     max_device_memory: str = None,
     device_target: str = "Ascend",
+    precision_mode: str = None,
```

Comment:
Why don't you use `init_train_env` from `mindone.env` as it already meets your needs? In this case, it will be easier to maintain `latte` and `dit` in sync.

Reply:
I agree. I wasn't aware of `init_train_env ` in `mindone`, but it's a good suggestion.